chain/db/table.go

```diff
@@ -27,3 +27,22 @@ type ValidatorVote struct {
 	BlockHeight      int64  `gorm:"primary_key;auto_increment:false"`
 	Voted            bool
 }
+
+type DataSource struct {
+	ID          int64 `gorm:"primary_key;auto_increment:false"`
+	Name        string
```

Comment:
Let's decorate everything that's not nullable `not null`. We will have better life with GraphQL that way, right @evilpeach ?

Reply:
Absolutely

Reply:
Already did